resources/views/mgt/facility/index.blade.php

```diff
@@ -469,10 +469,10 @@ class="fa fa-info-circle"></i>
             html += '<tr><td>' + resp.data[i].cid + '</td>'
             html += '<td data-sort-value="'+resp.data[i].lname + ', ' + resp.data[i].fname+'">'
             if (resp.data[i].isMentor == true) html += '<span class=\'label label-danger role-label\'>MTR</span> '
+            if (resp.data[i].isSupIns == true || (resp.data[i].rating>7&&resp.data[i].rating<11)) html += ' <span class=\'label label-danger role-label\'>INS</span> '
```

Comment:
Should probably make this if/elseif with INS having priority, we don't want to show both tags if for some reason someone has both roles.